oauth2_http/java/com/google/auth/oauth2/GoogleCredentials.java

```diff
@@ -233,6 +246,11 @@ public String getUniverseDomain() throws IOException {
     return this.universeDomain;
   }
 
+  @VisibleForTesting
```

Comment:
Please put detailed documentation here to indicate the purpose of this field and expected use by subclasses.

Reply:
SG